src/fastFood.css

```diff
@@ -0,0 +1,36 @@
+*{
```

Comment:
very good step, you also should add this border-box to ::before and ::after, this approach works fine when we are not using 3rd party component library. For 3rd party support you can consider this approach
:root {
  box-sizing: border-box;           1
}

*,
::before,
::after {
box-sizing:  inherit;
}
and for 3rd party component if needed u can use 
.third-party-components{
box-sizing: content-box;
}